src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1368,7 +1368,7 @@ MulticopterAttitudeControl::start()
 	_control_task = px4_task_spawn_cmd("mc_att_control",
 					   SCHED_DEFAULT,
 					   SCHED_PRIORITY_MAX - 5,
-					   1700,
+					   2200,
```

Comment:
Memory is already limited on the pixhawk for some configurations. Are these increases the absolute minimum you need?